src/components/ChatEntry.jsx

```diff
@@ -1,20 +1,73 @@
 import './ChatEntry.css';
+import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp.jsx';
+
+
+const ChatEntry = (props) => {
+  const likeButtonClicked = () => {
+    props.onLikeToggle(props.id);
+  };
```

Comment:
I like this pattern of sending just the `id` to `props.onLikeToggle` since it keeps all the state management and message object creation confined to `App`.